local-cluster/src/local_cluster.rs

```diff
@@ -63,6 +69,52 @@ use {
     },
 };
 
+pub fn build_tpu_quic_client(
+    cluster: &LocalCluster,
+) -> Result<TpuClient<QuicPool, QuicConnectionManager, QuicConfig>> {
```

Comment:
minor: is there no alias for this Quic version of the TpuClient?
if there is, let's re-use that, if not no need to add.

Reply:
as it turns out there is here: https://github.com/gregcusack/solana/blob/31d04a8d6f0550bf4dac7dab6cef34fb5e00483a/client/src/send_and_confirm_transactions_in_parallel.rs#L37
Could possibly make this more accessible and then use it everywhere its needed?

Reply:
yeah feel free to do that separately, but can probably just `pub` it or define in appropriate place.

Reply:
the problem is that the `QuicTpuClient` defined in that link above is using a different `TpuClient` than the one I need. It is using [this](https://github.com/gregcusack/solana/blob/68132a5930357f8155fe508b7083ea39bc361179/tpu-client/src/nonblocking/tpu_client.rs#L244) instead of what we need, which is [this](https://github.com/gregcusack/solana/blob/68132a5930357f8155fe508b7083ea39bc361179/client/src/tpu_client.rs#L26) `TpuClient`. Although this will change in the next PR. Since slowly going away from `solana_client/tpu_client` in favor of `tpu_client/`